docusaurus/docs/e2e-test-a-plugin/authentication.md

```diff
@@ -94,3 +94,28 @@ export default defineConfig<PluginOptions>({
   ]
 })
 ```
+
+## Managing users
+
+When a `user` is defined in a setup project like in the RBAC example above, `plugin-e2e` attempts to create the user using the Grafana HTTP API. This action requires elevevated permissions, so by default the server administrator credentials `admin:admin` will be used. If the end-to-end tests are targeting the [development environment](../get-started/set-up-development-environment.mdx) scaffolded with the `create-plugin`, this works fine but for other test environments the server adminstator password has likely been changed. In that case, you can provide the correct credentials by setting `grafanaAPICredentials` in the global options.
```

Comment:
nit: minor changes to try to make the sentences flow more easily due to a lot of use of use, using and user. 

```suggestion
When a `user` is defined in a setup project (like in the RBAC example above) `plugin-e2e` will use the Grafana HTTP API to create the user account. This action requires elevated permissions, so by default the server administrator credentials `admin:admin` will be used. If the end-to-end tests are targeting the [development environment](../get-started/set-up-development-environment.mdx) scaffolded with `create-plugin`, this will work fine. However for other test environments the server administrator password may be different. In that case, you can provide the correct credentials by setting `grafanaAPICredentials` in the global options.
```

Reply:
Much better, thanks!